modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/components/modal/ImportStructureModalContent.tsx

```diff
@@ -0,0 +1,168 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayAlert from '@clayui/alert';
+import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
+import {ClayInput} from '@clayui/form';
+import ClayModal from '@clayui/modal';
+import {sub} from 'frontend-js-web';
+import React, {useRef, useState} from 'react';
+
+const JSON_EXTENSION = '.json';
+
+export default function ImportStructureModalContent({
+	closeModal,
+}: {
+	closeModal: () => void;
+}) {
+	const [warning, setWarning] = useState(true);
+	const [jsonFile, setJsonFile] = useState<File | null>(null);
+	const fileInputRef = useRef<HTMLInputElement>();
```

Comment:
Can you initialize the fileInputRef with null?

```ts
const fileInputRef = useRef<HTMLInputElement | null>(null);
```

With this change we can discard the following `// @ts-ignore`

Reply:
done!